### PR TITLE
frum 0.1.1

### PR DESCRIPTION
**Pull request by TaKO8Ki**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
on linux

```
==> eval "$(/home/linuxbrew/.linuxbrew/Cellar/frum/0.1.1/bin/frum init)" && frum versions
Error: frum: failed
An exception occurred within a child process:
  Minitest::Assertion: Expected /2\.4\.0/ to match "  2.6.5".
```

File: Formula/frum.rb [hunk 1]
```diff
@@ -41,7 +41,7 @@ def install
     mkdir_p frum_dir/"versions/2.4.0"
     versions = shell_output("eval \"$(#{bin}/frum init)\" && frum versions").split("\n")
     assert_equal 2, versions.length
-    assert_match "2.4.0", versions[0]
-    assert_match "2.6.5", versions[1]
+    assert_includes versions, "  2.4.0"
```
Comment (chenrui333) [inline Formula/frum.rb:44]: can we remove the spaces? 
```suggestion
    assert_includes versions, "2.4.0"
```

Comment (TaKO8Ki) [reply Formula/frum.rb]: I removed spaces!

Comment (chenrui333) [reply Formula/frum.rb:44]: I see, I think it is fine then. Thanks!

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @TaKO8Ki!  
Contributor

**Pull request closed without merging**